.openpublishing.redirection.fundamentals.json

```diff
@@ -88,6 +88,14 @@
             "source_path_from_root": "/docs/fundamentals/code-analysis/style-rules/ide0074.md",
             "redirect_url": "/dotnet/fundamentals/code-analysis/style-rules/ide0054-ide0074"
         },
+        {
+            "source_path_from_root": "/docs/fundamentals/code-analysis/quality-rules/ide0160.md",
+            "redirect_url": "/dotnet/core/deploying/single-file/warnings/ide0160-ide0161.md"
+        },
+        {
+            "source_path_from_root": "/docs/fundamentals/code-analysis/quality-rules/ide0161.md",
+            "redirect_url": "/dotnet/core/deploying/single-file/warnings/ide0160-ide0161"
```

Comment:
```suggestion
            "redirect_url": "/dotnet/fundamentals/code-analysis/style-rules/ide0160-ide0161"
```

Reply:
Oops this was really sloppy of me. Thanks @Youssef1313!